src/pages/RegisterPage.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="register">
+    <CredentialsSection
+      v-if="currentPanel === 'CredentialsSection'"
+      @goToNextStep="goToNextStep"
+    />
+    <AdditionalDetailsSection v-else />
+  </div>
+</template>
+
+<script>
+import CredentialsSection from "../components/Register/CredentialsSection.vue";
+import AdditionalDetailsSection from "../components/Register/AdditionalDetailsSection.vue";
```

Comment:
How about the `@` alias? 👀 

```suggestion
import CredentialsSection from "@/components/Register/CredentialsSection.vue";
import AdditionalDetailsSection from "@/components/Register/AdditionalDetailsSection.vue";
```